oauth.md

```diff
@@ -75,14 +77,12 @@ define the scope of data access for your widget.
 
 **Notice**
 > If the user has already authorized your app, this step is skipped.  
-> At the moment, only pre-authorized partner applications are allowed, so this
-> step is always skipped.
 
 When the authorization succeeded, the user gets redirected to the `redirect_uri`
 with the access token as URI fragment:
 
 ```
-https://example.com#access_token=aaaaaa&expires_in=3600&token_type=bearer&auth_url=https%3A%2F%2Fapi-sandbox.allthings.me%2Fauth%2Fauthorize
+https://example.com#access_token=aaaaaa&expires_in=3600&token_type=bearer&state=color%3Agreen
```

Comment:
To be consistent with the request URL above, must contain the `theme` parameter.

Reply:
Also, shouldn't the state be still a JSON object?